mmv1/products/redis/api.yaml

```diff
@@ -190,3 +190,12 @@ objects:
           - :STANDARD_HA
         default_value: :BASIC
         input: true
+      - !ruby/object:Api::Type::Enum
+        name: transitEncryptionMode
+        description: |
+          The TLS mode of the Redis instance, If not provided, TLS is disabled for the instance.
+
+          - SERVER_AUTHENTICATION: Client to Server traffic encryption enabled with server authentcation
+        values:
+          - :SERVER_AUTHENTICATION
```

Comment:
We should add `:DISABLED` to this list as well so that users can specify it explicitly, I think. (as well as the docs)

Reply:
One more followup, can you add an update test going from :SERVER_AUTHENTICATION -> :DISABLED?

Reply:
We spoke offline (in chat)- this can be enabled but not disabled. We can use a customizediff to have Terraform handle both appropriately.

Reply:
API behavior is changed now, its not accepting the update for that attribute anymore. So schema attribute is set to ForceNew